src/main/kotlin/se/uulm/snowballr/backend/service/accessrules/ProjectAccessRule.kt

```diff
@@ -62,6 +63,25 @@ fun isProjectAdmin(projectMemberRepo: IProjectMemberTableRepo) = AccessRule<UUID
     projectAdmins.any { it.userId == requester.id }
 }
 
+/**
+ * Check whether the user is the last project admin of a specific project; otherwise, throws a [FailedPreconditionException].
```

Comment:
```suggestion
 * Check whether the user is not the last project admin of a specific project; otherwise, throws a [FailedPreconditionException].
```
We check whether the user is not the last project admin and only throw if they are the last project admin.

Reply:
You are right, good catch. I started this method the other way around and noticed I does not make sense and inverted the functionality without adapting the comment.